README.md

```diff
@@ -109,20 +110,36 @@ equation solver.
  prob1 = ODEProblem(f1,u0,tspan,p)
 
  σ = 0.01                         # noise, fixed for now
- t = collect(linspace(1,10,10))   # observation times
+ t = collect(1.:10.)   # observation times
  sol = solve(prob1,Tsit5())
-
+ priors = [Normal(1.5, 1)]
  randomized = VectorOfArray([(sol(t[i]) + σ * randn(2)) for i in 1:length(t)])
  data = convert(Array,randomized)
+ 
+ using CmdStan #required for using the Stan backend
+ bayesian_result_stan = stan_inference(prob1,t,data,priors)
+
+ bayesian_result_turing = turing_inference(prob1,Tsit5(),t,data,priors)
+ 
+ using DynamicHMC
+ bayesian_result_hmc = dynamichmc_inference(prob1, Tsit5(), t, data, priors)
+
+ bayesian_result_abc = abc_inference(prob1, Tsit5(), t, data, priors)
+```
+### Using save_idxs for using latent variables 
```

Comment:
Using save_idxs for using latent variables  -> Using save_idxs to declare observables?